tests/models/plants/test_plants_model.py

```diff
@@ -21,11 +23,17 @@ def test_PlantsModel__init__(plants_data, flora):
     assert len(plants_model.flora) == len(flora)
     assert len(plants_model.communities) == plants_data.grid.n_cells
 
-    # Check the canopy has been initialised and updated
-    assert "layer_heights" in plants_data
-    assert "leaf_area_index" in plants_data
-    assert plants_data["layer_heights"].sum() == (30 + 20 + 10) * 4
-    assert plants_data["leaf_area_index"].sum() == 3 * 4
+    # Check the canopy has been initialised and updated with some simple test sums
+    expected_layers = [
+        ("layer_heights", (30 + 20 + 10) * 4),
+        ("leaf_area_index", 3 * 4),
+        ("layer_fapar", (0.4 + 0.2 + 0.1) * 4),
+        ("layer_absorbed_irradiation", 1000 * 4),
+    ]
+
+    for layer_name, layer_sum in expected_layers:
+        assert layer_name in plants_data
+        assert plants_data[layer_name].sum() == layer_sum
```

Comment:
Because this is a floating-point comparison you should use `pytest.approx`, otherwise you might get spurious failures.

Reply:
Yup - I'd already imported `numpy` to use `np.allclose` in one of those tests, don't know why I didn't use it for the others.